canonical-web-and-design/cn.ubuntu.com
# Engage pages and takeovers

## Done
- Scss linting config consistent with ubuntu.com. so a lot of file formatting.
- Move engage pages to discourse
- Removed html files
- JS homepage takeovers

## QA
- Go to https://cn-ubuntu-com-475.demos.haus and check the homepage takeovers are correct
  - Active takeovers are showing
  - Image sizes are correct
  - They display even if you break the takeovers.json endpoint

- You can see the list of takeovers https://cn-ubuntu-com-475.demos.haus/takeovers
- Go to https://cn-ubuntu-com-475.demos.haus/engage
- Click on list of engage pages, make sure they show and forms work

Edit content: https://discourse.ubuntu.com/t/engage-pages-and-takeovers-for-cn-ubuntu-com/19117

## Reminder before merge
Replace thank-you page absolute links to `https://cn.ubuntu.com`

## Issue / Card

#452
